testing/testing_page.html

```diff
@@ -63,7 +65,86 @@ <h1>Widgetastic About Modal</h1>
       </div>
     </div>
   </div>
+
+  <div>
```

Comment:
Wasn't able to get the checkable part of this tree working correctly.

I've tested in the CFME application's implementation of the checkable bootstrap tree, but haven't unit tested via this testing_page.

Reply:
Is this correct? I would assume you give the JS an empty div and a dictionary of data and it will populate the DOM for you.

Reply:
No, but you are. I pulled the reference from Patternfly's site instead of trying to pull from MIQ.  Working better now.